client/my-sites/posts/post-share.jsx

```diff
@@ -71,7 +71,7 @@ const PostSharing = React.createClass( {
 					'is-active': ( this.state.skipped.indexOf( connection.keyring_connection_ID ) === -1 )
 				} ) }
 			>
-				<SocialLogo icon={ connection.service }/>
+				<SocialLogo icon={ connection.service === 'google_plus' ? 'google-plus' : connection.service }/>
```

Comment:
👍 Must have missed this one, there was drift between the social logos repo and the component in Calypso. See #9399

Just fyi that any logo name from social logos will be in kebab case.